.github/workflows/ci.yml

```diff
@@ -18,21 +17,67 @@ on:
   workflow_dispatch:
 
 jobs:
-  examples:
-    name: Build site
+  build-unminified-site:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Build Site with Hugo and Audit"
+        uses: danielfdickinson/build-audit-action-hugo-dfd@v0.2
+        with:
+          upload-site-as: unminified-site
+          use-lfs: false
+          hugo-extended: true
+          hugo-version: 0.92.2
+  check-links:
+    needs: build-unminified-site
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Setup Hugo
-        uses: peaceiris/actions-hugo@v2
+      - name: Run DFD Hugo Check Links
+        uses: danielfdickinson/link-check-action-hugo-dfd@v0.2
```

Comment:
I recently found this functionality in with [zola check](https://www.getzola.org/documentation/getting-started/cli-usage/#check).

Reply:
I also would prefer to use zola as well, but there are a significant amount of shortcodes and templates to port over...

Reply:
This took longer than expected, but these checks are performed by `htmltest`